app/src/main/java/com/hammer/talkbbokki/domain/usecase/TopicUseCase.kt

```diff
@@ -16,4 +16,7 @@ class TopicUseCase @Inject constructor(
         return repository.getTopicList(level)
             .flowOn(dispatcher)
     }
+
+    fun getTodayViewCnt(): Flow<Int> = repository.getTodayViewCnt()
+    fun setTodayViewCnt(isReset: Boolean = false): Flow<Int> = repository.setTodayViewCnt(isReset).flowOn(dispatcher)
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)